src/store/store.js

```diff
@@ -0,0 +1,16 @@
+import { configureStore } from '@reduxjs/toolkit'
+import { actionTypes, firebaseReducer } from 'react-redux-firebase'
+import { firestoreReducer } from 'redux-firestore'
+
+export const store = configureStore({
+  reducer: {
+    firebaseReducer,
+    firestoreReducer,
+  },
+  middleware: (getDefaultMiddleware) =>
+    getDefaultMiddleware({
+      serializableCheck: {
+        ignoredActions: [actionTypes.LOGIN, actionTypes.AUTH_LINK_ERROR],
+      },
```

Comment:
I'm not sure if you need this, but you can leave it right now.

Reply:
🤷